packages/react-components/react-list-preview/src/components/ListItem/ListItem.types.ts

```diff
@@ -1,17 +1,22 @@
+import { Checkbox } from '@fluentui/react-checkbox';
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { ListItemActionEvent } from '../../events/ListItemActionEvent';
 
 export type ListItemSlots = {
-  root: Slot<'div'>;
+  root: NonNullable<Slot<'li', 'div'>>;
+  checkmark?: Slot<typeof Checkbox>;
 };
 
 /**
  * ListItem Props
  */
-export type ListItemProps = ComponentProps<ListItemSlots> & {};
+export type ListItemProps = ComponentProps<ListItemSlots> & {
+  value?: string | number;
+  // eslint-disable-next-line @nx/workspace-consistent-callback-type -- using custom event here with no data
+  onAction?: (e: ListItemActionEvent) => void;
```

Comment:
nit: would be nice to find a way to keep this handler signature consistent with the callback types

Reply:
I'll bring this up in the next team meeting 🙂